recipe/patches/0006-add-ldl-to-libstdc___la_LDFLAGS.patch

```diff
@@ -0,0 +1,22 @@
+From b8b0bcd9d97f2dc59c566e0a836d3dc70ddd947b Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Mon, 15 Jul 2024 19:16:05 +1100
+Subject: [PATCH 6/6] add `-ldl` to libstdc___la_LDFLAGS
+
+---
+ libstdc++-v3/src/Makefile.am | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/libstdc++-v3/src/Makefile.am b/libstdc++-v3/src/Makefile.am
```

Comment:
You need to edit `Makefile.in` file too. `.in` file is generated from `.am` file.

Reply:
(cross-posted something similar in https://github.com/conda-forge/ctng-compilers-feedstock/pull/142#discussion_r1680453673 )

Reply:
Ah OK; I thought the `.am` gets generated from the `.in` file?

Reply:
`*.am`/`*.ac` files are inputs for autotools ("ac": autoconf, "am": automake) to generate configure scripts and the template makefiles (`*.in`); configure scripts will then generate the final makefiles from the `*.in` files.